src/api.js

```diff
@@ -1,12 +1,30 @@
-const BASE_URL = "https://bootcamp-api.codeit.kr/api/sample";
+const BASE_URL = "https://bootcamp-api.codeit.kr/api";
 
 export async function getUser() {
-  const response = await fetch(`${BASE_URL}/user`);
+  const response = await fetch(`${BASE_URL}/sample/user`);
   const body = await response.json();
   return body;
 }
+
 export async function getFolder() {
-  const response = await fetch(`${BASE_URL}/folder`);
+  const response = await fetch(`${BASE_URL}/sample/folder`);
   const body = await response.json();
   return body;
 }
+
+export async function getFolders() {
+  const userId = 1;
+  const response = await fetch(`${BASE_URL}/users/${userId}/folders`);
+  const body = await response.json();
+  return body;
+}
+
+export async function getFolderLinks(id) {
+  const userId = 1;
+  const folderId = id === "" ? null : +id;
```

Comment:
```suggestion
  const folderId = id ? +id : null;
```

js에서 ""은 falsy한 값이라서 위와 같이 작성하실 수 있습니다!